src/main/scala/com/scalaSparkSkillValidation/utils/DataGenerator.scala

```diff
@@ -0,0 +1,43 @@
+package com.scalaSparkSkillValidation.utils
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import com.scalaSparkSkillValidation.models._
+
+import java.sql.{Timestamp, _}
+
+object DataGenerator {
+
```

Comment:
Ok, let's simulate real project here:
1. so we do not generate data - in main part we assume data exists in Tables, Storage - ok?
2. so anything you need to create Storage, Tables you need to do it in test case: similar to the one we done on a interview just in test space! OK? If you do not understand please ask me in the issue or in the PR!
3. communication is one of the things I check here